openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -48,18 +51,83 @@ def replace_missing_key(match):
         pattern = r"\{([^}]*)\}"
         return re.sub(pattern, replace_missing_key, template)
 
-    def process(self, context):
-        for instance in context:
-            # Check if instance is a review by checking its family
-            # Allow a match to primary family or any of families
-            families = set(
-                [instance.data["family"]] + instance.data.get("families", [])
+    def get_settings_from_context(self, context):
+        """Get settings from context if they're different.
+
+        As we sometimes have wrong loaded settings in this integrator,
+        but have the good ones loaded in the context, it's better to use them.
+        """
+        kitsu_note = (
+            context.data["project_settings"]
+            .get("kitsu", {})
+            .get("publish", {})
+            .get(self.__class__.__name__, {})
+        )
+        if (
+            kitsu_note.get("set_status_note") != self.set_status_note,
+            kitsu_note.get("note_status_shortname")
+            != self.note_status_shortname,
+            kitsu_note.get("status_change_conditions")
+            != self.status_change_conditions,
+        ):
+            self.set_status_note = kitsu_note["set_status_note"]
```

Comment:
I think each condition must be tested to each parameter instead of doing it as a bunch because you might want to keep only one and not overriding them all.

```suggestion
            if kitsu_note.get("set_status_note") != self.set_status_note:
                 self.set_status_note = kitsu_note["set_status_note"]
```

Reply:
I unchained my (heart) conditions in https://github.com/Tilix4/OpenPype/pull/78/commits/791c25c98fc65300094d0d3d420157492f984c2a